_pages-partials/getting-started/05-controls.html.twig

```diff
@@ -0,0 +1,226 @@
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Spawn</p>
+
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key">I</kbd>
+                            </div>
+                            <div class="control">
+                                {% include '_includes/mouse.html.twig' with { pressed: 'right' } %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Shoot</p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key enter" data-align="right">
+                                    enter
+                                </kbd>
+                            </div>
+                            <div class="control">
+                                {% include '_includes/mouse.html.twig' with { pressed: 'left' } %}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">Jump</p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key tab" data-align="left">
+                                    tab
+                                </kbd>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4 mb-4">
+                    <div class="kb-control-group">
+                        <p class="title">
+                            Identify / Lock On
+                            <abbr title="with Guided Missile flag">*</abbr>
+                            player
+                            <span class="sr-only">with Guided Missile flag</span>
+                        </p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key">I</kbd>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-sm-6 col-lg-4">
+                    <div class="kb-control-group">
+                        <p class="title">Drop Flag</p>
+                        <div class="controls">
+                            <div class="control">
+                                <kbd class="c-keyboard__key space" data-align="center">space</kbd>
```

Comment:
Can this become a Twig macro to generate KB keys instead? It'd be nice to generate keys consistently (especially if they have "shift values").